### ISSUE TITLE
Update wiki page "Http client"

### ISSUE THREAD

**Issue by KindDragon** [Microsoft/cpprestsdk#716]
Please add to this wiki page https://github.com/Microsoft/cpprestsdk/wiki/HTTP-Client some useful information

You can just copy information from this page https://msdn.microsoft.com/en-us/library/jj950081.aspx

**Comment by EKenny31** [comment]
Yeah, it's ridiculous there's no documentation. It's hard to trust something that doesn't appear to be well maintained.
